httpclient5/src/main/java/org/apache/hc/client5/http/impl/auth/GGSSchemeBase.java

```diff
@@ -73,7 +72,8 @@
     }
 
     private static final Logger LOG = LoggerFactory.getLogger(GGSSchemeBase.class);
-
+    private static final String NO_TOKEN = "";
+    private static final String KERBEROS_SCHEME = "HTTP";
```

Comment:
This name is semantically wrong! It is a service class and not a Kerberos scheme. I also don't understand why make a difference between `null` and `#isEmpty()` with a static final field.